.ci/build.sh

```diff
@@ -5,7 +5,14 @@ SCRIPT_DIR="$(dirname "$0")"
 set -exuo pipefail
 source "$SCRIPT_DIR/common"
 
-docker build -t $CI_IMAGE_URI --build-arg IMAGE_REPOSITORY=${IMAGE} --build-arg IMAGE_TAG=${COMMIT} "$SCRIPT_DIR/.."
+docker buildx build \
```

Comment:
this is making it hard to test as it's trying to push to a docker registry.
Perhaps we should use the same thing we do on the other locals, ex:
https://github.com/openebs/dynamic-localpv-provisioner/blob/develop/Makefile.buildx.mk
Which *I think* had allowed me to do local builds previously, though I'd have to test that again as well

Reply:
I was able to do a test with a simple

```
docker run --rm --network=host registry:2
```

Because I have tried several ways, and I was unable to achieve multi arch build that fit into this setup that we have

Reply:
I was able to do it now by setting a single platform and doing load rather than push:
 docker buildx build \
-  --push --pull --no-cache \
+    --load \
   --platform="${CI_IMAGE_PLATFORMS}" \
   -t "$CI_IMAGE_URI" \
   --build-arg "IMAGE_REPOSITORY=${IMAGE}" \
   --build-arg "IMAGE_TAG=${COMMIT}" "$SCRIPT_DIR/.."
 
-docker pull "$CI_IMAGE_URI"
+#docker pull "$CI_IMAGE_URI"

Reply:
CI is now passing here.
Before merging do you to change this so that locally (or at least by default) we build with the params above and with only the local platform?